### PR TITLE
Fixes platform services

### PR DESCRIPTION
**Pull request by dianaolives**

### CODE REVIEW

File: content/change-logs/platform-services/2023-12-06_changing-data-in-the-access-mapping-section.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06T15:49:04.056Z
+title: Changing data in the access mapping section
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_changing-data-in-the-access-mapping-section.md:3]:
```suggestion
title: Fixed issue with Save button in Access mapping section
```

File: content/change-logs/platform-services/2023-12-06_fixed-issue-from-invalid-pagesize-or-currentpage-parameter.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Fixed issue from invalid pageSize or currentPage parameter
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_fixed-issue-from-invalid-pagesize-or-currentpage-parameter.md:3]:
```suggestion
title: Fixed issue with invalid parameters
```

File: content/change-logs/platform-services/2023-12-06_fixed-issues-with-page-redirection.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Fixed issues with page redirection
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_fixed-issues-with-page-redirection.md:3]:
```suggestion
title: Fixed issues with redirection to the SSO server
```

File: content/change-logs/platform-services/2023-12-06_fixed-issues-with-the-ericsson-dcp-sms-provider.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06T16:20:15.159Z
+title: Fixed issues with the Ericsson DCP SMS provider
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+product_area: Platform services
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_fixed-issues-with-the-ericsson-dcp-sms-provider.md:10]: label should be "SMS microservice". Not sure if this is already available

File: content/change-logs/platform-services/2023-12-06_http-code-issue-fixed.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: HTTP code issue fixed
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_http-code-issue-fixed.md:3]:
```suggestion
title: HTTP error issue fixed
```

File: content/change-logs/platform-services/2023-12-06_improved-custom-alarm-property.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06T16:24:32.165Z
+title: Improved custom alarm property
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_improved-custom-alarm-property.md:3]:
```suggestion
title: Fixed redundant audit log entries creation
```

File: content/change-logs/platform-services/2023-12-06_improved-password-strength-indicator.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06T15:34:00.835Z
+title: Improved password strength indicator
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_improved-password-strength-indicator.md:3]:
```suggestion
title: Fixed issue with password strength indicator
```

File: content/change-logs/platform-services/2023-12-06_improved-proof-of-possession-process.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Improved Proof of Possession process
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_improved-proof-of-possession-process.md:3]:
```suggestion
title: Fixed issues with Proof of Possession process
```

File: content/change-logs/platform-services/2023-12-06_improved-unpacking-of-versioned-applications.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06T15:55:16.702Z
+title: Improved unpacking of versioned applications
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_improved-unpacking-of-versioned-applications.md:3]:
```suggestion
title: Fixed issue with unpacking versioned applications
```

File: content/change-logs/platform-services/2023-12-06_improved-unsecured-login-option.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Improved unsecured login option
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_improved-unsecured-login-option.md:3]:
```suggestion
title: Fixed unsecured login option
```

File: content/change-logs/platform-services/2023-12-06_log-files-stored-in-platform-can-now-be-downloaded.md [hunk 11]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06T16:00:12.648Z
+title: Log files stored in platform can now be downloaded
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_log-files-stored-in-platform-can-now-be-downloaded.md:3]:
```suggestion
title: Log files of devices can now be downloaded
```

File: content/change-logs/platform-services/2023-12-06_messaging-service-requests-time-out-quickly.md [hunk 12]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Messaging Service requests time out quickly
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_messaging-service-requests-time-out-quickly.md:3]:
```suggestion
title: Fixed issues with Messaging Service requests time out
```

File: content/change-logs/platform-services/2023-12-06_no-unequal-distribution-of-notifications.md [hunk 13]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: No unequal distribution of notifications
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_no-unequal-distribution-of-notifications.md:3]:
```suggestion
title: Fixed issue with unequal distribution of notifications
```

File: content/change-logs/platform-services/2023-12-06_password-reset-when-user-logs-in-with-oai-secure.md [hunk 14]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Password reset when user logs in with OAI-Secure
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_password-reset-when-user-logs-in-with-oai-secure.md:3]:
```suggestion
title: Password reset on OAI-Secure login
```

File: content/change-logs/platform-services/2023-12-06_refreshing-session-tokens-with-two-factor-authentication.md [hunk 15]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Refreshing session tokens with two-factor-authentication
```
Comment (BeateRixen) [inline content/change-logs/platform-services/2023-12-06_refreshing-session-tokens-with-two-factor-authentication.md:3]:
```suggestion
title: Fixed issues with refreshing session tokens
```

**Comment by BeateRixen** [comment]
@sarangovindarajan @martin-sag Please review the first bunch of fixes for the 2024 release notes. Especially please check if the titles are ok which have been added by Diana.

File: content/change-logs/platform-services/cumulocity-10-18-151-0-simple-type-name-accepted-as-type-filter.md [hunk 16]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Simple type name accepted as type filter
```
Comment (sagscmi) [inline content/change-logs/platform-services/cumulocity-10-18-151-0-simple-type-name-accepted-as-type-filter.md:3]:
```suggestion
title: Simple type name accepted as type filter for Notifications 2.0 subscriptions
```

File: content/change-logs/platform-services/cumulocity-10-18-185-0-fixed-issues-with-messaging-service-requests-time-out.md [hunk 17]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Fixed issues with Messaging Service requests time out
```
Comment (sagscmi) [inline content/change-logs/platform-services/cumulocity-10-18-185-0-fixed-issues-with-messaging-service-requests-time-out.md:3]:
```suggestion
title: Fixed issues with Messaging Service request timeouts
```

File: content/change-logs/platform-services/cumulocity-10-18-230-0-delete-notifications-to-specific-managed-objects.md [hunk 18]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: DELETE notifications to specific managed objects
```
Comment (sagscmi) [inline content/change-logs/platform-services/cumulocity-10-18-230-0-delete-notifications-to-specific-managed-objects.md:3]:
```suggestion
title: DELETE notifications for specific managed objects with Notifications 2.0
```

**Review by sagscmi — CHANGES_REQUESTED** [review]
Some suggested changes to title, to make it more obvious when the issue is about Notifications 2.0 vs. Realtime Notifications (both under the REST API component)

File: content/change-logs/platform-services/cumulocity-10-18-489-0-fixed-unsecured-login-option.md [hunk 19]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Fixed unsecured login option
```
Comment (sarangovindarajan) [inline content/change-logs/platform-services/cumulocity-10-18-489-0-fixed-unsecured-login-option.md:3]: Fixed information disclosure issues with unsecured login option endpoint.

File: content/change-logs/platform-services/ui-c8y-10-18-369-0-improved-tooltip-to-copy-ssl-command.md [hunk 20]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Improved tooltip to copy SSL command
```
Comment (sarangovindarajan) [inline content/change-logs/platform-services/ui-c8y-10-18-369-0-improved-tooltip-to-copy-ssl-command.md:3]: Improved tooltip usability in Proof of possession

File: content/change-logs/platform-services/ui-c8y-10-18-497-8-redirection-to-management-tenant-removed.md [hunk 21]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Redirection to Management tenant removed
```
Comment (sarangovindarajan) [inline content/change-logs/platform-services/ui-c8y-10-18-497-8-redirection-to-management-tenant-removed.md:3]: Addressed the redirection issues for support user

File: content/change-logs/platform-services/ui-c8y-10-18-500-17-improved-sso-configuration-page.md [hunk 22]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2023-12-06
+title: Improved SSO configuration page
```
Comment (sarangovindarajan) [inline content/change-logs/platform-services/ui-c8y-10-18-500-17-improved-sso-configuration-page.md:3]: Improved SSO configurations

**Review by sarangovindarajan — CHANGES_REQUESTED** [review]
Please refer to my comments..

**Review by martin-sag — APPROVED** [review]

**Review by sagscmi — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**